### PR TITLE
Documentation Inconsistency: The level of described change is patch not minor. Plz Update Index.rst

### PR DESCRIPTION
**Pull request by Alagts**
Comparing https://semver.org/ with https://docs.typo3.org/m/typo3/guide-installation/main/en-us/Minor/Index.html#minor shows that the word "minor" is not correct. It has to be clarified as a patch level update. If I see this correctly the update for a minor change is not described in the documentation since the wrong minor article is the bugfix upgrade article. Please correct me if I missed something.

### CODE REVIEW

File: Documentation/Minor/Index.rst [hunk 1]
```diff
@@ -3,32 +3,32 @@
 .. _minor:
 
 =============
```
Comment (brotkrueml) [inline Documentation/Minor/Index.rst:5]:
```suggestion
====================
```

File: Documentation/Minor/Index.rst [hunk 2]
```diff
@@ -3,32 +3,32 @@
...
-Minor upgrade
+Patch/Bugfix upgrade
 =============
```
Comment (brotkrueml) [inline Documentation/Minor/Index.rst:7]:
```suggestion
====================
```

File: Documentation/Minor/Index.rst [hunk 3]
```diff
@@ -3,32 +3,32 @@
...
 
-What are minor upgrades
+What are patch/bugfix upgrades
 =======================
```
Comment (brotkrueml) [inline Documentation/Minor/Index.rst:10]:
```suggestion
==============================
```

**Review by brotkrueml — APPROVED** [review]

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
@Alagts thanks for catching this

**Pull request merged**